packages/connect/src/protocol-connect/handler-factory.spec.ts

```diff
@@ -51,51 +53,89 @@ describe("createHandlerFactory()", function () {
     },
   } as const;
```

Comment:
suggestion: if we want the best of both worlds (having the type contract and also a constant), we can do

```suggestion
  } satisfies ServiceType;
```

Reply:
Yep, forgot this one, the other two have it already. We have to be cautious with `satisfies`, we promise compat with TypeScript 4.1. We'll have to add ts-compat checks like in protobuf-es if we start marking more widespread use.